src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,116 @@
 package core.basesyntax;
 
 public class MyHashMap<K, V> implements MyMap<K, V> {
```

Comment:
Don't begin class or method implementation with an empty line. Remove the empty line after the class declaration.